R/downloadFileAsDataFrame.R

```diff
@@ -0,0 +1,50 @@
+'data.world-r
+Copyright 2017 data.world, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+
+You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+implied. See the License for the specific language governing
+permissions and limitations under the License.
+
+This product includes software developed at data.world, Inc.(http://www.data.world/).'
+
+#' Download the input file into an R data frame
+#' @param connection the connection to data.world
+#' @param dataset the "agentid/datasetid" for the dataset against which to execute the query
+#' @param fileName the filename as listed in the data.world dataset (only csv file is supported for now)
+#'
+#' @export
+downloadFileAsDataFrame <- function(connection, dataset, fileName) {
+  UseMethod("downloadFileAsDataFrame")
+}
+
+#' @export
+downloadFileAsDataFrame.default <- function(connection, dataset, fileName) {
+  print("nope.")
+}
+
+#' @export
+downloadFileAsDataFrame.data.world <- function(connection, dataset, fileName) {
+  if (!endsWith(fileName, ".csv")) {
```

Comment:
yup . @bryonjacob and i have discussed some potential usage of data package . Probably will be included in the next release / PR